cosmwasm/contracts/crosschain-registry/src/state.rs

```diff
@@ -0,0 +1,22 @@
+use cosmwasm_std::Addr;
+use cw_storage_plus::{Item, Map};
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+
+// CONTRACT_ALIAS_MAP is a map from a contract alias to a contract address
+pub const CONTRACT_ALIAS_MAP: Map<&str, String> = Map::new("contract_alias_map");
+
+// CHAIN_TO_CHAIN_CHANNEL_MAP is a map from a source<>destination chain pair to its respective channel id
+pub const CHAIN_TO_CHAIN_CHANNEL_MAP: Map<(&str, &str), String> =
```

Comment:
Did you experiment with using `Chain` and `ChannelId` here? If they implement `FromStr` we could also call create them by calling `"osmosis".into()`

Reply:
I did, and I am running into some issues (specifically stuck at having to implement a PrimaryKey to Chain in order to use load)

Reply:
Ahh then there's no point. It's not like it adds so much clarity or we need many methods on that type